packages/web/components/table/asset-balances.tsx

```diff
@@ -253,6 +257,12 @@ export const AssetBalancesTable: FunctionComponent<{
 
   return (
     <div className="w-full">
+      <ExternalLinkModal
+        url={externalUrl ?? ""}
+        isOpen={Boolean(externalUrl)}
+        onRequestClose={useCallback(() => setExternalUrl(null), [])}
```

Comment:
Any reason why we are using a `useCallback`?  It becomes an overhead if the component does not perform reference checks which I believe `ExternalLinkModal` doesn't.

Reply:
I'll admit, I thought React only re-rendered children when props changed, but it appears it only does that when we use memo. Otherwise it always re-renders children.

https://react.dev/reference/react/memo